### DISCUSSION TITLE
How I can change themes?

### DISCUSSION THREAD

**Discussion by NomanJaved** [nasirkhan/laravel-starter#355] [Q&A] [answered]
How I can change the frontend or backend theme?

Which themes is used for admin and frontend themes?

appricate your efforts.

**Comment by nasirkhan** [comment, marked as answer]
Thanks for your question. 

There is no straightway to change the themes. In the frontend I used Tailwind and for backend, I used Bootstrap5 based theme CoreUI. Other details are also mentioned in the readme file: https://github.com/nasirkhan/laravel-starter/blob/main/README.md 

You may develop a feature for changing the theme, and submit a pull request. This may help others in future.

**Comment by NomanJaved** [comment]
is there any tutorial for how to make new modules. 

Like 

New menu 
New controller
New view
New model
New Route

So to get better understanding to dig into quickly.

**Comment by nasirkhan** [comment]
All the new additions can be done as per Laravel core documentation. You may follow that and create a new controller view model and so on. 
There is another approach following the project which is Module. Each module is like a separate Laravel package and structured as a basic Laravel application. I suggest you to check the source code first to understand how this modules works.
